gelf.go

```diff
@@ -202,6 +202,14 @@ func TimeKey(value string) Option {
 	})
 }
 
+// DynamicLevel set dynamic logging level.
+func DynamicLevel(level zap.AtomicLevel) Option {
```

Comment:
Please, rename the option to LevelAtomic. This option should use the same pattern as other options such as Level and LevelString.